{"author": {}, "body": "[deleted]", "created_utc": 1586241766.0, "distinguished": null, "id": "fmo19u1", "is_submitter": false, "link_id": "t3_fuy9mr", "parent_id": "t1_fmo0p9i", "score": 1, "stickied": false, "subreddit_id": "t5_2sjnq", "downs": 0, "total_awards_received": 0, "edited": 1586241967.0, "name": "t1_fmo19u1", "collapsed": false, "permalink": "/r/TechNewsToday/comments/fuy9mr/no_5g_didnt_cause_the_coronavirus_pandemic/fmo19u1/"}